tests/installation/partitioning_raid.pm

```diff
@@ -42,7 +42,12 @@ sub addpart {
     send_key $cmd{donotformat};
     send_key "tab";
 
-    send_key_until_needlematch 'partition-selected-raid-type', 'down';
+    if ($size == 300 and get_var('UEFI')) {
```

Comment:
I didn't find a better solution, on the other hand there was exactly the same thing before in addraid():
`if ($step == 2 and get_var("UEFI")) {`
and I moved this logic into run(), but IMO not possible in this case. I will provide a comment.